core/tests/test_utils.py

```diff
@@ -486,6 +497,12 @@ def put_json(self, url, payload, csrf_token=None, expect_errors=False,
         json_response = self.testapp.put(
             str(url), data, expect_errors=expect_errors)
 
+        # Testapp takes in a status parameter which is the expected status of
+        # the response. However this expected status is verified only when
+        # expect_errors=False. For other situations we need to explicitly check
+        # the status.
+        # Reference URL:
```

Comment:
Add a blank line above this one.

Use a URL that refers to the specific line that shows this behaviour, like the one I pointed out to you in chat. E.g. start from here and find the do_request() stuff and the line in there that has the "if not expect_errors" check: https://github.com/Pylons/webtest/tree/25bfadac1d3288fb5f52b36081c636e836694372

Also this reference URL should be replicated in the other two docstrings.

Reply:
Ah sorry you've done that :) LGTM!

Reply:
Ah I was just doing this. Linked it to the "if not expect_errors" line.